app/src/main/java/org/wikipedia/page/leadimages/LeadImagesHandler.java

```diff
@@ -208,53 +208,40 @@ private void updateCallToAction() {
         if (!AccountUtil.isLoggedIn() || getLeadImageUrl() == null || !getLeadImageUrl().contains(URL_FRAGMENT_FROM_COMMONS) || getPage() == null) {
             return;
         }
-        GalleryItem[] galleryItem = {null};
-        String[] title = {null};
-        disposables.add(ServiceFactory.getRest(getTitle().getWikiSite()).getMedia(getTitle().getConvertedText())
-                .flatMap(gallery -> {
-                    List<GalleryItem> list = gallery.getItems("image");
-                    for (GalleryItem item : list) {
-                        if (getPage() != null && item.getFilePage().contains(StringUtil.addUnderscores(getPage().getPageProperties().getLeadImageName()))) {
-                            galleryItem[0] = item;
-                            title[0] = item.getFilePage().equals(Service.COMMONS_URL) ? item.getTitles().getCanonical() : UriUtil.getTitleFromUrl(item.getFilePage());
-                            return MediaHelper.INSTANCE.getImageCaptions(title[0]);
-                        }
-                    }
-                    return null;
-                })
+        String imageTitle = "File:" + getPage().getPageProperties().getLeadImageName();
```

Comment:
@dbrant Are we sure this will not create a problem given that some files are "File:" and some are File: "?

Reply:
Not sure what the second case means. Can you give an example?

Reply:
Sorry. Not an issue. got confused with something else.